ceph/ceph_admin/__init__.py

```diff
@@ -138,32 +139,47 @@ def set_tool_repo(self, repo=None):
             for node in self.cluster.get_nodes():
                 node.exec_command(sudo=True, cmd=cmd)
 
-    def set_cdn_tool_repo(self):
+    def set_cdn_tool_repo(self, release=None):
```

Comment:
Kindly add `doc string` for the new parameter.

Reply:
Done.